pkg/model/last_run.go

```diff
@@ -0,0 +1,30 @@
+package model
+
+import "time"
+
+// LastBackupRun stores the last run times for both full and incremental backups.
+type LastBackupRun struct {
+	// Last time the Full backup was performed.
+	Full *time.Time
+	// Last time the Incremental backup was performed.
+	Incremental *time.Time
+}
+
+func NewLastRun(lastFullBackup *time.Time, lastIncrBackup *time.Time) LastBackupRun {
```

Comment:
Consider renaming to `MakeLastBackupRun` or `CreateLastBackupRun`, since it is not a good practice to return a value for `New` constructors.